### PR TITLE
Update all READMEs

### LINKED ISSUE #272: Documentation needs to be updated to reflect changes in the latest codebase.
The READMEs need to be updated and reviewed for conformity with the latest changes to the server, settings.cfg, and dbtool. 

Also, this is a good opportunity to interface with the community and take feedback for streamlining/improving the docs for clarity and simplified use, especially for those without substantial coding experience.

There are several READMEs that need to be updated (these are complete in #418):
- [x] [./README.md](https://github.com/Segs/Segs/blob/7450f2abffc9effaf0cb7748d0568e746d1cc214/README.md) - the root directory readme and first thing users see when viewing our GitHub. This should be geared towards a developer audience looking to compile, BUT needs to provide quick links and clear instructions on where to find our installation and setup guides (and readme)
- [x] [./Projects/CoX/docs/README.md](https://github.com/Segs/Segs/blob/7450f2abffc9effaf0cb7748d0568e746d1cc214/Projects/CoX/docs/README.md) - This readme will appear in the SEGS program (out) directory. It should be geared towards setting up and running the server. It should be clear and concise, and focus on the most common methods for install (don't need to cover every possible configuration, just the most common ones)
- [x] [./Projects/CoX/Utilities/dbtool/README.md](https://github.com/Segs/Segs/blob/0901a04dd4c05d8d92d78ef43265d4d90da45b51/Projects/CoX/Utilities/dbtool/README.md) - an argument might be made for deleting this, but I think it may be valuable to have. At bare minimum we should leave it, and include a link inside to the setup readme in the program directory.
- [x] [./Projects/CoX/Utilities/piggtool/README.md](https://github.com/Segs/Segs/blob/b68162ac97d3b41aefc8f8dc0ba44484f3798682/Projects/CoX/Utilities/piggtool/README.md) - an argument might be made for deleting this as well, but I think it may be valuable to have, if it's updated with real instructions. At bare minimum we should leave it, and include a link inside to the setup readme in the program directory.

### PR DESCRIPTION
**Pull request by broxen**
Feedback greatly appreciated!

Closes #272 

Additions/modifications proposed in this pull request:
- [./README.md](https://github.com/broxen/Segs/blob/update-docs/README.md) - the root directory readme and first thing users see when viewing our GitHub. Updated to provide quick links for INSTALL
- [./Projects/CoX/docs/README.md](https://github.com/broxen/Segs/blob/update-docs/Projects/CoX/docs/README.md) - This readme will appear in the SEGS program (out) directory. Updated to focus on SEGSAdmin tool workflow
- [./Projects/CoX/Utilities/dbtool/README.md](https://github.com/broxen/Segs/blob/update-docs/Projects/CoX/Utilities/dbtool/README.md) - updated with basic command line arguments
- [./Projects/CoX/Utilities/piggtool/README.md](https://github.com/broxen/Segs/blob/update-docs/Projects/CoX/Utilities/piggtool/README.md) - updated with basic command line arguments
- Renamed `authors.txt` to AUTHORS.md
- Renamed `faq.md` to FAQ.md
- Renamed `LICENSE` to LICENSE.md for consistency and file type association in Windows

### CODE REVIEW

**Comment by broxen** [comment]
whoops forgot to [skip-ci]

File: Projects/CoX/docs/README.md [hunk 1]
```diff
@@ -30,176 +29,109 @@ SEGS requires:
...
-   If you would like to run the client in windowed mode for debugging purposes, add these optional instructions: `-fullscreen 0 -screen 1024 768`.
+   To setup your copy of SEGS click each of the following buttons and follow the on screen instructions:
    
-   When you want to launch the client, you will need to use this shortcut.
+   1. **Generate Config File**
 
-7. Continue Server Setup
-   
-   The client is now ready for play, but continue the server setup by following along below.
+      Clicking this button will automatically determine your IP and configure the settings.cfg file SEGS needs to run. It will ask you to enter a name for your server.
```
Comment (broxen) [inline Projects/CoX/docs/README.md]: SEGSAdmin does not automatically determine your IP, but rather asks for input. I need to add information on obtaining the IP address here.

File: Projects/CoX/docs/README.md [hunk 2]
```diff
@@ -30,176 +29,109 @@ SEGS requires:
...
-   - `listen_addr` -- This is the IP of the server that clients will connect to, followed by `:7002` which is the default listening port. If running your server locally this CANNOT be 127.0.0.1 (localhost) and must point to your actual IP address on the network. The port never changes.
-   - `location_addr` -- This is the IP of the server where the clients will receive from the server, followed by `:7002` which is the default listening port. If running your server locally this CANNOT be 127.0.0.1 (localhost) and must point to your actual IP address on the network. The port never changes.
-   - `max_players` -- The maximum number of concurrent connections allowed by the server.
-   - `max_character_slots` -- This is the default number of character slots allowed for each account.
+   In your CoX Client folder, locate and run `CoX-Auth.exe` to set the IP address to that of your server. If running your server locally this CANNOT be 127.0.0.1 (localhost) and must point to your actual IP address on the network. For Windows users, a guide to determining your IP can be found here: https://support.microsoft.com/en-us/help/15291/windows-find-pc-ip-address
```
Comment (broxen) [inline Projects/CoX/docs/README.md]: Add section on using ipconfig instead?

Comment (broxen) [reply Projects/CoX/docs/README.md]: or leverage #419 if implemented in time for Version 0.5.0

File: Projects/CoX/docs/README.md [hunk 3]
```diff
@@ -30,176 +29,109 @@ SEGS requires:
...
+   | Access Level    | Permissions                          |
+   | -------------   | ------------------------------------ |
+   | 0               | basic user with no special authority |
+   | 1               | game master, has limited authority   |
+   | 2-8             | reserved for future use              |
+   | 9               | administrator with full access       |
 
 
-CONFIGURATION
+INSTALL & CONFIGURE THE COX CLIENT
```
Comment (miles200) [inline Projects/CoX/docs/README.md]: Should we move this above the SEGSAdmin section above? As users need to have installed the CoX client for the .pigg files for stage 3 in server setup in SEGSAdmin

Comment (miles200) [inline Projects/CoX/docs/README.md]: Should we move this above the SEGSAdmin section above? As users need to have installed the CoX client for the .pigg files for stage 3 in server setup in SEGSAdmin

**Comment by dracc** [comment]
Suggestion for addition in ./README.md;
- OPTIONAL: MySQL - By default SEGS uses a local SQLite database, but optionally may be configured to use MySQL instead. http://www.mysql.org/downloads/mysql

File: docs/AUTHORS.md [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 mobbyg          <mobbyg AT segs DOT io>
 nemerle         (http://www.nemerle.eu/) <nemerle AT segs DOT io>
 
+
 SEGS Contributors
-=================
+======
 Armingleggings
 Ashleigh
 cameron-justice
```
Comment (dracc) [inline docs/AUTHORS.md]: These are now on a single row. Would be beautiful with a single row per name. :+1:

Comment (miles200) [reply docs/AUTHORS.md]: Have you tried a double space after each line, should push it to a new row

Comment (broxen) [reply docs/AUTHORS.md]: I'm sorry I don't understand. Each name is on it's own line already?

Or is this a Windows line break issue?

Comment (dracc) [reply docs/AUTHORS.md]: Open the file here on github and you'll see they're all on a single line. :)

Comment (broxen) [reply docs/AUTHORS.md]: Ahh makes sense. I'll check LICENSE.md for similar issues.

**Comment by broxen** [comment]
Add dracc and crescens to Authors.md

**Comment by broxen** [comment]
bah.. forgot to skip ci again. I'm terrible.

Feedback welcome!

**Comment by broxen** [comment]
My Readme changes to the logo have broken the badges. I need to figure out why before this merges.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-![logo](docs/segs-medallion-med.png)
+<img align="right" src="docs/segs-medallion-med.png">
```
Comment (broxen) [inline README.md]: This is breaking the badges for some reason

Comment (dracc) [reply README.md]: I believe you need a trailing slash inside the tag, I.E. it needs to end with `/>` instead of just `>`. 🙂

**Comment by broxen** [comment]
Fixed. Not sure what the hell

File: README.md [hunk 6]
```diff
@@ -8,42 +10,50 @@ SEGS - Super Entity Game Server
...
 SEGS is the Super Entity Game Server. It is a free, open source (under the terms of the BSD License), cross-platform, and object-oriented suite of protocols and services designed to interface with popular super hero themed MMORPG clients. SEGS is written in C++ and facilitates common MMORPG functionality such as account and character storage to a database, account and character retrieval from a database, secure client authentication, client movement in a virtual world, and client chat.
 
-For legal information, please see [LICENSE](./LICENSE).
+For legal information, please see [LICENSE.md](./LICENSE.md).
 
-For a list of SEGS authors and contributors, please see [Authors.txt](./docs/Authors.txt).
+For a list of SEGS authors and contributors, please see [AUTHORS.md](./docs/AUTHORS.md).
 
+**:exclamation: LOOKING FOR INSTALLATION INSTRUCTIONS? :exclamation:**
+> NOTICE: This guide is for COMPILING SEGS FROM SOURCE. If you're looking for help installing and configuring SEGS from a release package, please view [the install README.md here](./Projects/CoX/docs/README.md)
```
Comment (broxen) [inline README.md]: Remove "install" from before README.md as it's usage is confusing.

File: README.md [hunk 7]
```diff
@@ -8,42 +10,50 @@ SEGS - Super Entity Game Server
...
 REQUIREMENTS AND NOTES
 ------
 
 Below are the utilities and libraries you'll need to compile SEGS in any environment. While it may be possible to use another toolset, the C++ Tool Chain below is the only one officially supported by the SEGS team. These packages are available for both Linux or Windows, via your distro's package manager, or via the links below:
 
-   - QT >5.8 - A cross platform application framework utilized heavily by SEGS. https://www.qt.io/download-qt-for-application-development  
-   - CMake >3.6 - CMake is the cross-platform make utility. It generates makefiles for multiple platforms. https://cmake.org/download/
-   - Git - A version control system for tracking changes in computer files and coordinating work on those files among multiple people. https://git-scm.com/download  
-   - _OPTIONAL:_ PostGreSQL - By default SEGS uses a local SQLite database, but optionally may be configured to use PostGreSQL instead. http://www.postgresql.org/download/
+   - **QT >5.8** - A cross platform application framework utilized heavily by SEGS. https://www.qt.io/download-qt-for-application-development
```
Comment (broxen) [inline README.md]: Maybe should be 5.9+?

**Comment by broxen** [comment]
Logo should probably shrink further as it currently cuts off all README text on mobile. Maybe 70% it's current size?

**Pull request merged**